soh/src/overlays/actors/ovl_Boss_Ganon/z_boss_ganon.c

```diff
@@ -11,6 +11,8 @@
 #include "assets/scenes/dungeons/ganon_boss/ganon_boss_scene.h"
 
 #include "soh/frame_interpolation.h"
+#include "soh/Enhancements/boss-rush/BossRush.h"
```

Comment:
All these bossrush includes in the boss overlays can be removed now that boss rush most stuff is VBified